### DISCUSSION TITLE
diary/2023/07/13/how-to-make-a-lecture/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#145] [Comments]
# diary/2023/07/13/how-to-make-a-lecture/

키워드 : 인프런 강의 올리기, 인프런 강의 만들기, 인프런 강의 제작, 온라인 강의 만들기, 온라인 강의 제작 인프런에 PM을 위한 데이터 리터러시 강의를 23년 2월에 올렸습니다. 강의를 만들면서 어떻게 좋은 강의를 만들어야 할까?를 계속 고민했습니다 추후 강의를 만들고자 하는 분들을 위해 이 글을 작성합니다 예상 독자 인프런에서 강의를 만들까 고민하고...

https://zzsza.github.io/diary/2023/07/13/how-to-make-a-lecture/

**Comment by CODe5753** [comment]
평소 온라인 강의 준비에 관심이 많았는데, 노하우 공유 감사합니다 😃

**Reply by zzsza** [reply]
온라인 강의 준비에 관심이 많으셨군요..! 인프런에서 올려주신 강의 만드는 경험기도 한번 꼭 봐보셔요!

**Comment by Ssojux2** [comment]
오오 재밌게 읽었습니다! 공유 감사합니다 :)

**Reply by zzsza** [reply]
읽어주셔서 감사합니다!! :)

**Comment by joesiheon496** [comment]
한개의 강의를 찍기위한 과정을 볼 수 있어서 재미있었습니다.
항상 자기 자신에게 질문하시는 모습이 멋있어요!

**Reply by zzsza** [reply]
강의 찍는 과정에서 해야하는 일이 생각보다 많아서, 이렇게 정리해두면 누군가에게 도움이 될 것 같아 남겨봤어요! 읽어주셔서 감사합니다

**Comment by jws1837** [comment]
오오. 강의 하나 찍는 것이 얼마나 힘든지 느꼈습니다. 대단해요!

**Comment by MyType9591** [comment]
아낌없이 주시는 성윤님... 감동입니다.. 감사합니다!

**Comment by durumee** [comment]
섬세한 경험 나눠주셔서 감사합니다 👍
